### ISSUE TITLE
Exception in alba_id_t: (Failure "attempting to use a resource from a pool which is being finalized")

### ISSUE THREAD

**Issue by domsj** [openvstorage/alba#775]
```
Aug 07 11:52:56 ctrl-221-3 alba[16810]: 2017-08-07 11:52:50 830713 +0200 - ctrl-221-3 - 16810/0000 - alba/maintenance - 8139446807 - info - Exception in alba_id_t: (Failure "attempting to use a resource from a pool which is being finalized"); backtrace:; Raised at file "format.ml", line 241, characters 41-52; Called from file "format.ml", line 482, characters 6-24
Aug 07 11:52:56 ctrl-221-3 alba[16810]: 2017-08-07 11:52:50 830729 +0200 - ctrl-221-3 - 16810/0000 - alba/maintenance - 8139446808 - info - Exception in alba_id_t: (Failure "attempting to use a resource from a pool which is being finalized"); backtrace:; Raised at file "format.ml", line 241, characters 41-52; Called from file "format.ml", line 482, characters 6-24
```

@pploegaert saw this in a maintenance process on his test env
and IIRC I also saw this on a proxy on the nuvolat setup

still to be further investigated

**Comment by pploegaert** [comment]
fwiw (old alba version)

```
root@ctrl-221-3:/var/tmp# alba version
ee-1.4.2
git_revision: "tags/1.4.2-0-g093692a"
git_repo: "Not available"
compile_time: "03/05/2017 13:21:51 UTC"
machine: "172396896fd1 4.4.0-36-generic x86_64 x86_64 x86_64 GNU/Linux"
model_name: "Intel(R) Xeon(R) CPU E31220 @ 3.10GHz"
compiler_version: "4.03.0"
is_alba_test: false
dependencies:
arakoon_client                    1.9.17        Arakoon client
bz2                                0.6.0        [n/a]
cmdliner                          v1.0.0        Declarative definition of command line interfaces
core                       [unspecified]        
ctypes                            0.11.3        Combinators for binding to C libraries without writing any C.
ctypes.foreign                    0.11.3        Dynamic linking of C functions
kinetic-client                     0.0.6        Kinetic client
lwt                                3.0.0        Lightweight thread library for OCaml (core library)
lwt.unix                           3.0.0        Unix support for Lwt
oUnit                              2.0.0        Unit testing framework
ocplib-endian                        1.0        Optimised functions to read and write int16/32/64 from strings and bigarrays
ppx_deriving.enum                    4.1        [@@deriving enum]
ppx_deriving.show                    4.1        [@@deriving show]
ppx_deriving_yojson                  3.0        [@@deriving yojson]
redis                              0.3.3        Ocaml bindings for Redis
rocks                              0.3.0        Rocksdb binding
sexplib                    [unspecified]        
snappy                             0.1.0        Bindings to snappy compression library
ssl                                0.5.3        OCaml bindings to libssl
tiny_json                          1.1.3        A small Json library from OCAMLTTER
uri                                1.9.2        RFC3986 URI parsing library
yojson                             1.3.3        JSON parsing and printing (successor of json-wheel)
```

**Comment by domsj** [comment]
seen again by @jeroenmaelbrancke 
`@domsj After we changed the alba manager config on the global proxies (fragment cache) we see following logging:`
```2017-08-09 02:36:46 538539 -0400 - NY1SRV0001 - 6205/0000 - alba/proxy - 1271052 - info - Exception in alba_id_t: (Failure "attempting to use a resource from a pool which is being finalized")```

**Comment by domsj** [comment]
@jeroenmaelbrancke also told me this kept on being logged, up to 247 times per minute

**Comment by domsj** [comment]
might be related to #777?

**Comment by domsj** [comment]
fixed by #789
